lib/ukatomic/Config.uk

```diff
@@ -0,0 +1,8 @@
+menuconfig LIBUKATOMIC
+	bool "ukatomic: Atomic operations"
+	default y
+	help
+	  This option enables the atomic operations library, implemented
```

Comment:
This message can be improved a bit, as currently it provides either redundant information, or information not useful to the user. I explain:

> implemented for all supported architectures

Architecture support should be addressed by Kconfig dependencies, ie libraries not supported by a given architecture normally don't appear to the menu.

> This library is required by other libraries and applications, and should be enabled by default.

This is redundant as you already enable this by setting `default y`.

It's better to provide something as simple as `Atomic operations` or a more elaborate description like `Architecture-specific implementation of atomic operations. The set of operations is the same as the one provided by GCC's memory model aware atomic built-ins.`